Makefile

```
@@ -154,13 +155,21 @@ otelcol:
 	go generate ./...
 	$(MAKE) build-binary-internal
 
+.PHONY: otelcol-licenses
+otelcol-licenses: cmd/otelcol/third-party
+
+cmd/otelcol/third-party:
+	@echo creating third_party directory with licenses + reciprical source
+	cd ./cmd/otelcol && go-licenses save . --save_path=third-party
+	chmod +w $(find ./cmd/otelcol/third_party -type d)
```

Had some issues running this due to makefile target and paths not matching up (third_party/third-party). This worked for me:

```suggestion
.PHONY: otelcol-licenses
otelcol-licenses:
	@echo creating third_party directory with licenses + reciprical source
	go-licenses save ./cmd/otelcol --save_path=./cmd/otelcol/third-party --force
	chmod -R u+w ./cmd/otelcol/third-party
```

Was the `go.mod` to get rid of messages like
> main module (go.opentelemetry.io/collector) does not contain package go.opentelemetry.io/collector/cmd/otelcol/third-party/github.com/hashicorp/consul/api

I get those as well but they seem harmless. Would be nice not to have them but 🤷

---

Ok, so I fixed the typo (sorry bout that), and I've been upgrading.  Seems we still get issues here so I'm toying around with some solutions.